src/main/java/org/fundacionjala/at13/spaceinvaders/Spaceship.java

```diff
@@ -15,4 +21,37 @@ public Spaceship(final int pX, final int pY, final int llimit, final int ulimit)
     public Bullet shoot() {
         return new Bullet(this);
     }
+
+    /**
+     * Increase 30 to score value ofSpaceship..
+     */
+    public void increaseScore() {
+        this.score += VALUE;
+    }
+
+    /**
+     * @return a score value ofSpaceship.
+     */
+    public int getScore() {
+        return this.score;
+    }
+
+    /**
+    * @return Ask if spacehip has more than 3 lives
+    */
+    public void isStillAlive() {
```

Comment:
Wouldn't it be better if this method is applied in spaceElement? considering that an alien can also die.